ts/Core/Interaction.ts

```diff
@@ -853,10 +853,14 @@ extend(Chart.prototype, /** @lends Chart.prototype */ {
                             isNumber(dataExtremes.dataMax)
                         ) {
                             panningState.startMin = Math.min(
-                                dataExtremes.dataMin, panningState.startMin
+                                pick(series.options.threshold, Infinity),
+                                dataExtremes.dataMin,
+                                panningState.startMin
                             );
                             panningState.startMax = Math.max(
-                                dataExtremes.dataMax, panningState.startMax
+                                pick(series.options.threshold, Infinity),
```

Comment:
This means that when `series.options.threshold` is `null` or undefined, `startMax` becomes `Infinity`. And the consequence can be see here: https://jsfiddle.net/highcharts/zupgLn59/1/

I think it should be `pick(series.options.threshold, -Infinity)`